assets/javascripts/render.js

```diff
@@ -187,6 +187,8 @@ function renderModuleTable(container, response) {
     container.appendChild(E('table', [thead, tbody],
         {id: 'results', 'class': 'table table-striped'}));
 
+    let summary = {};
```

Comment:
If you're already using the scoped variable declarations, why not using `const` here? Same counts for `modules` in the loop below.

Reply:
Agree, Only I was consistent with the previous code that used let.